docs/guide/static-deploy.md

```diff
@@ -47,15 +47,15 @@ You may configure the port of the server py passing `--port` flag as an argument
 ```json
 {
   "scripts": {
-    "serve": "vite preview --port 8080"
+    "preview": "vite preview --port 8080"
   }
 }
 ```
 
 Now the `preview` method will launch the server at http://localhost:8080.
 
 ::: tip NOTE
-If you change the script name from `serve` to `preview`, you may run into issues with some package managers due to the way they handle [Pre & Post scripts](https://docs.npmjs.com/cli/v7/using-npm/scripts#pre--post-scripts).
+If you have a `view` command, you may run into issues with the `preview` command in some package managers due to the way they handle [Pre & Post scripts](https://docs.npmjs.com/cli/v7/using-npm/scripts#pre--post-scripts).
```

Comment:
Do we still need this note? It could still be useful, but I don't know if it is that valuable to justify its space in the docs.

Reply:
I would vote not to have them in the docs.

Reply:
done 👍🏼